src/Application.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * Slim Framework (https://slimframework.com)
+ *
+ * @license https://github.com/slimphp/Slim-Console/blob/0.x/LICENSE.md (MIT License)
+ */
+
+declare(strict_types=1);
+
+namespace Slim\Console;
+
+use Symfony\Component\Console\Application as SymfonyApplication;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Throwable;
+
+class Application extends SymfonyApplication {
+    private const VERSION = '0.1';
+
+    public function __construct()
+    {
+        parent::__construct('Slim Console', self::VERSION);
```

Comment:
@ABGEO07 I added the name/version here instead of keeping it in the bin file. I think it's going to be easier to keep track of/update version this way

Reply:
@l0gicgate  :+1: Good decision. Agree with you.